apps/balancer-tools/src/app/poolsimulator/(components)/AnalysisPage.tsx

```diff
@@ -27,10 +35,21 @@ export default function Page() {
     setCurrentTabTokenByIndex(tokensSymbol.indexOf(target.innerText));
   }
 
+  const poolTypes = [initialData?.poolType, customData?.poolType];
+  const poolsHaveDifferentPurpose =
```

Comment:
`#note` if 0 or 2 match, this list won't have length 1 👍

Reply:
I can leave a comment about this, but actually that is what we want. If 0 or 2 match is bc none matches is bc both are for non-stable and if both matches is bc both are stable.

Reply:
Yea I understood it, just left a note to self/others reviewing the PR 👍